src/main/java/dan200/computercraft/client/render/MonitorTextureBufferShader.java

```diff
@@ -117,8 +117,6 @@ private static boolean load()
                 ComputerCraft.log.warn( "Problems when linking monitor shader: {}", log );
             }
 
-            GL20.glDetachShader( vertexShader, program );
-            GL20.glDetachShader( fragmentShader, program );
```

Comment:
`glDeleteShader` only 'marks' a shader for deletion. If it's still attached to a program, it won't actually be deleted. So this was fine, actually. Unless `GlStateManager.deleteShader` already does that? Too lazy to check.

Reply:
These two lines were throwing `GL_INVALID_OPERATION`, and I gave up. 

And I just realised my arguments were swapped 🤦. Who needs a type system I guess?

Reply:
Oh dear, I didn't notice either :P.